src/main/java/frc/robot/subsystems/scoring/AimerIORoboRio.java

```diff
@@ -214,13 +217,13 @@ public void updateInputs(AimerIOInputs inputs) {
         }
 
         appliedVolts = MathUtil.clamp(appliedVolts, -12.0, 12.0);
-        if (!motorDisabled || override) {
+        if (enableOutput || override) {
```

Comment:
I think this should be `enableOutput && override`, not sure why we did it this way

Reply:
It turns out that I was wrong - this needs to be an OR gate otherwise no output happens 💯